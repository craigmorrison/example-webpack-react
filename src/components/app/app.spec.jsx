import { render } from '@testing-library/react';
import App from './app';

describe('app', () => {
  it('matches the snapshot', () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });
});
