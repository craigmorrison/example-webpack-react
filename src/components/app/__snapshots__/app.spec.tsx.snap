// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app matches the snapshot 1`] = `
<body>
  <div>
    <header
      role="banner"
    >
      <h1>
        Title
      </h1>
    </header>
    <main>
      <h2>
        Section title
      </h2>
      <p>
        Hello
      </p>
    </main>
    <footer
      role="contentinfo"
    >
      Legal Stuff
    </footer>
  </div>
</body>
`;
